### PR TITLE
Restricting visualization to current project if possible

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| Infra |
|**_Affected Component_**| fprime-util visualize |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**| nasa/fprime/#2354  |
|**_Has Unit Tests (y/n)_**| n/a |
|**_Builds Without Errors (y/n)_**| y  |
|**_Unit Tests Pass (y/n)_**| y |
|**_Documentation Included (y/n)_**| na/ |

---
## Change Description

Uses a projects inferred name to sub-select a topology to visualize when running with sub-topolgies!

## Rationale

Subtopologies make multiple `AppAi.xml` files but the user just wants to visualize the composite topology not the sub topologies.  Visualizing just the subtopology can be done in the subtopology's module.

### CODE REVIEW

**Comment by LeStarch** [comment]
Also made it launch the webbrowser like the GDS does so that one doesn't need to know where to go look!

File: src/fprime/fpp/visualize.py [hunk 1]
```diff
@@ -126,6 +130,7 @@ def run_fprime_visualize(
     config = {"SOURCE_DIRS": [str(viz_cache.resolve())]}
     app = construct_app(config)
     try:
+        webbrowser.open(f"http://localhost:{parsed.gui_port}")
```
Comment (thomas-bc) [inline src/fprime/fpp/visualize.py:133]: Should this come after app.run() ? Not that any browser would load that quickly... but just because?

Comment (LeStarch) [reply src/fprime/fpp/visualize.py:133]: No, because app.run will block until an interrupt.

Comment (LeStarch) [reply src/fprime/fpp/visualize.py:133]: To guarantee ordering, we'd need to spin-off a thread.  I am not sure that is worth it.

Comment (thomas-bc) [reply src/fprime/fpp/visualize.py:133]: Ohh right right, sounds good

**Review by thomas-bc — APPROVED** [review]

**Pull request merged**